src/olympia/users/tasks.py

```diff
@@ -139,6 +146,9 @@ def sync_blocked_emails(batch_size=BATCH_SIZE, **kw):
 
                 email_blocks = [SuppressedEmail(email=record[3]) for record in batch]
                 SuppressedEmail.objects.bulk_create(email_blocks, ignore_conflicts=True)
+                count += len(batch)
```

Comment:
maybe we want some logging every batch to show progress?

Reply:
I don't think it's necessary, at least not yet. Only added the logging here because the download part can take some time to run and without any logs it can be confusing and made me wonder if something was wrong with the code. I added the count at the end to just verify that we have synced "something". The syncing is really fast due to the batching so it wouldn't add much value IMO.